src/app/excel/excel-table/excel-table.component.html

```diff
@@ -2,43 +2,63 @@
              [currentProjectName]="this.store.currentProject?.name"></app-menubar>
 <p-toast position="top-center"></p-toast>
 <p-card header="Podsumowanie dostępności">
-  <ng-template [ngIf]="isLoaded && declarableSprint">
+  <ng-template [ngIf]="currentSprint && sprintSummary">
     <div class="ui-g" id="availability-table">
-      <div class="ui-g-8">
-        <div class="ui-g">
-          <div class="ui-g-4">
-            <h5>Nazwa sprintu:</h5>
+      <div class="ui-g-12">
+        <div class="ui-g-12">
+          <div class="ui-g-1">
+            <p-button icon="pi pi-arrow-left"
+                      class="left"
+                      (click)="onPrevClick()"
+                      [disabled]="!sprintNavigationStatus.prevAvailable">
+            </p-button>
           </div>
-          <div class="ui-g-6">
-            <span>{{ declarableSprint.name }}</span>
-          </div>
-        </div>
-        <div class="ui-g">
-          <div class="ui-g-4">
-            <h5>Czas trwania:</h5>
+
+          <div class="ui-g-10 d-flex justify-content-center">
+            <div class="ui-g">
+              <div class="ui-g-4">
```

Comment:
Myślę, że jakby tu dać `ui-g-6` (a także w linii 25), to miałoby szanse być bardziej wyśrodkowane. Albo `ui-g-5` dla wszystkich divów tutaj.